src/views/lighttable.c

```diff
@@ -180,6 +180,8 @@ typedef struct dt_library_t
 
   int32_t collection_count;
 
+  gboolean culling_prefetched; // culling has already prefetch the current displayed images?
+
```

Comment:
I don't think this is needed, I've added `slots_changed` params, which indicate when the images shown have changed. It serve the same thing.
Note that I may have forgot places where to set it to false...